### ISSUE TITLE
Wrong positioning of \vec in subscript at sizes 12pt or less

### ISSUE THREAD

**Issue by isaacsb** [stipub/stixfonts#205]
Since at least release 2.10, `\vec` (also `\overrightarrow` with single character) when used in subscript is too much to the left at font sizes 12pt or less. Same results with `lualatex` and `xelatex` (using `unicode-math`). Haven't tried with Word or plain TeX.

It does not happen with `\overleftarrow`.

-------
Example:
```latex
\documentclass{standalone}
\usepackage{unicode-math}
\setmathfont{STIX Two Math}
\newcommand\test[1]{\fontsize{#1}{18}\selectfont {#1}pt: $\partial_{\vec{v}} f$}
\begin{document}
\test{8} \quad \test{9} \quad \test{10} \quad \test{11} \quad \test{12} \quad \test{13} \quad \test{14} \quad \test{15}
\end{document}
```

Using version 2.02 b142:
![v2 02-b142](https://user-images.githubusercontent.com/3248154/115959778-2c857100-a50e-11eb-95c0-040f87299dd0.png)
[v2.02-b142.pdf](https://github.com/stipub/stixfonts/files/6369620/v2.02-b142.pdf)

Using version 2.12 b168a (also 2.10+):
![v2 12-b168a](https://user-images.githubusercontent.com/3248154/115959794-3e671400-a50e-11eb-8299-30ec60239f05.png)
[v2.12-b168a.pdf](https://github.com/stipub/stixfonts/files/6369621/v2.12-b168a.pdf)

**Comment by tiroj** [comment]
Thanks. Will review possible causes for this issue and plan a fix for either v2.20 or an intermediate bug fix release. Strange that it is only affecting some sizes.

**Comment by firai** [comment]
By the way, this doesn't seem to be an issue with Word:
![Small Vec Word](https://user-images.githubusercontent.com/23555/116029249-6d56c480-a68b-11eb-9a5b-2a6bb830e5cb.png)
1st line: 11 pt.
2nd line: 7 pt.

(Produced with STIX Two Math 2.12 b168a and Word 16 [a.k.a. Office 365])

**Comment by tiroj** [comment]
Will check for related MATH table settings, but if this is not happening in MS Word then possibly the issue is in the TeX flavours?

**Comment by davidmjones** [comment]
> Will check for related MATH table settings, but if this is not happening in MS Word then possibly the issue is in the TeX flavours?

Yes.  This is related to #162.  By default, the unicode-math package used by both XeTeX and luaTeX turns on the ssty1 feature for any font sizes between 7 and 10pt.  [Sorry for not twigging on to the relevance of that to this issue sooner.]

**Comment by tiroj** [comment]
Yikes! So this is something that probably will end up getting fixed in the v2.20 work, as a free rider on improved ssty glyph support.
